README.es.md

```diff
@@ -164,17 +175,33 @@ del primer bloque.
 
 ```c++
 // Correcto
-if(!error)
+if (!error)
 {
     return success;
 }
 
 // Incorrecto
-if(!error)
+if (!error)
+    return success;
+
+// Incorrecto
+if (!error) return success;
+```
+
+El paréntesis de apertura de la lista de cláusulas debe encontrarse espaciado de la palabra reservada `if`.
```

Comment:
Creo que mejora mucho la legibilidad de los condicionales, y en la mayoría de ejemplos lo estáis usando así. Dejarlo por escrito es bien.

Reply:
Completamente de acuerdo